recipe/meta.yaml

```diff
@@ -152,3 +152,4 @@ about:
 extra:
   recipe-maintainers:
     - jcrist
+    - consideratio
```

Comment:
@jacobtomlinson @TomAugspurger @martindurant for example could perhaps be willing to help with conda-forge maintenance chores?

Reply:
Happy to be included